dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -69,6 +69,9 @@ def create
     )
     return head :bad_request unless section.persisted?
 
+    # Add all coteachers specified in params
+    params[:add_instructor_emails]&.each {|instructor_email|  SectionInstructor.create_section(section, instructor_email)}
```

Comment:
Nit: renaming to follow the pattern of other params being nouns rather than verbs

```suggestion
    params[:instructor_emails]&.each {|instructor_email|  SectionInstructor.create_section(section, instructor_email)}
```